student.rb

```diff
@@ -0,0 +1,11 @@
+require './person'
+class Student < Person
+  def initialize(classroom)
+    super(age)
+    @classroom = classroom
+  end
```

Comment:
- [ ] Kindly notice that you should send the person constructor parameters in the student constructor parameters.

```ruby
def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
end
```